api/tests/Feature.FormTemplates.UnitTests/Endpoints/AssignEndpointTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Feature.FormTemplates.AssignTemplates;
+using Feature.FormTemplates.Specifications;
+using Microsoft.EntityFrameworkCore;
+using NSubstitute.ReturnsExtensions;
+using Vote.Monitor.Domain.Entities.ElectionRoundAggregate;
+using Vote.Monitor.Domain.Entities.ElectionRoundFormTemplateAggregate;
+
+namespace Feature.FormTemplates.UnitTests.Endpoints;
+
+public class AssignEndpointTests
+{
+    private readonly IReadRepository<ElectionRound> _electionRoundRepository;
+    private readonly IReadRepository<FormTemplateAggregate> _formTemplateRepository;
+    private readonly IRepository<ElectionRoundFormTemplate> _electionRoundFormTemplateRepository;
+    private readonly AssignTemplates.Endpoint _endpoint;
+
+    public AssignEndpointTests()
+    {
+        _electionRoundRepository = Substitute.For<IReadRepository<ElectionRound>>();
+        _formTemplateRepository = Substitute.For<IReadRepository<FormTemplateAggregate>>();
+        _electionRoundFormTemplateRepository = Substitute.For<IRepository<ElectionRoundFormTemplate>>();
+        _endpoint = Factory.Create<Endpoint>(
+            _electionRoundRepository,
+                                    _formTemplateRepository,
+                                _electionRoundFormTemplateRepository);
+    }
+    
+    [Fact]
+    public async Task DeleteRangeAsync_Should_Return_NoContent()
```

Comment:
a better name will be `ShouldUnasignFormTemplates_WhenRequestDoesNotContainAssignedTemplates`